config/seata-config-core/src/main/java/org/apache/seata/config/ConfigurationCache.java

```diff
@@ -115,6 +76,9 @@ public Configuration proxy(Configuration originalConfiguration) throws Exception
                     }
                     if (null == wrapper
                             || (null != defaultValue && !Objects.equals(defaultValue, wrapper.lastDefaultValue))) {
+                        if (DATA_ID_CACHED.add(rawDataId)) {
+                            originalConfiguration.addConfigListener(rawDataId, this);
```

Comment:
Why do need to listen on every key?

Reply:
The hierarchical structure of the cache is below that of the config; not getting the configuration is meaningless for listening purposes.

Reply:
ok